lib/main/server.js

```diff
@@ -0,0 +1,50 @@
+var http = require('http');
+var url = require('url');
+var Scrapper = require('./scrapper.js');
+
+function Server() {
+
+  this.start = function() {
+    http.createServer(function(request, response) {
+      var urlParams = url.parse(request.url, true);
+      if (request.method === 'GET' && urlParams.pathname === '/I/want/title/') {
+        var addresses = (typeof(urlParams.query.address) == 'string') ? [urlParams.query.address] : urlParams.query.address;
```

Comment:
It is better to use strict comparison (===) instead of comparison with conversion (==) unless highly there is a real need.